app/cdap/components/Snackbar/Components/Transition/types.ts

```diff
@@ -14,17 +14,9 @@
  * the License.
  */
 
-import { makeStyles } from '@material-ui/core';
-
-export const useStyles = makeStyles({
-  snackBarDiv: {
-    padding: '10px',
-    display: 'block',
-    border: '1px solid #E97567',
-    boxShadow: '-3px 4px 15px rgba(68, 132, 245, 0.25)',
-    height: '148px',
-    width: '401px',
-    bottom: '10% !important',
-    backgroundColor: '#fff',
-  },
-});
+export interface ITransitionProps {
+  handleClose: () => void;
+  isSuccess: boolean;
+  actionType: string;
```

Comment:
can we rename this actionType key to something more appropriate plz, if required

Reply:
Done